cookbooks/crystal_preferred_orientation_olivine_fraters_billen_2021/cpo_pole_figure_config.toml

```
@@ -0,0 +1,15 @@
+base_dir = "path/to/where/you/run/aspect/input"
+experiment_dirs = ["out_drex_olivineA/"]
+compressed = false
+
+[pole_figures]
+    elastisity_header = false #not yet implemented
```

Shouldnt this be:
```suggestion
    elasticity_header = false #not yet implemented
```
Or is this actually written like that in the plotting package?